docker-compose-openmrs.yml

```diff
@@ -31,6 +31,9 @@ services:
       traefik.http.routers.openmrs.middlewares: openmrs-spa-redirectregex
       traefik.http.middlewares.openmrs-spa-redirectregex.redirectregex.regex: https://${O3_HOSTNAME}/openmrs/spa
       traefik.http.middlewares.openmrs-spa-redirectregex.redirectregex.replacement: https://${O3_HOSTNAME}/openmrs/spa/home
+      #=====================================================backup================================================================
+      restic-compose-backup.volumes: true
+      restic-compose-backup.volumes.include: "${OPENMRS_CONFIG_CHECKSUMS_PATH:-openmrs-config-checksums}"
```

Comment:
@enyachoke , I think we should backup more than just this volume by default.
I don't see volume declaration for OpenMRS complex obs for instance.

Reply:
@rbuisson  I have included person images and complex obs